arangograph/notebooks.md

```diff
@@ -59,3 +69,83 @@ at anytime. The notebook's phase is updated accordingly.
 
 1. Select the notebook that you want to remove from the **Notebooks** tab.
 2. Click the **Delete notebook** button.
+
+## Getting Started notebook
+
+To get a better understanding of how to interact with your ArangoDB database
+cluster, use the ArangoGraph Getting Started template.
+The ArangoGraph Notebook automatically connects to the ArangoDB service
+endpoint, so you can immediately start interacting with it.
+
+1. Log in into the notebook you have created.
+2. Open the file browser and navigate to the `jovyan` directory.
+3. Select the `GettingStarted.ipynb` file.
+
+## ArangoGraph Magic Commands
+
+A list of the available magic commands you can interact with.
+Single line commands have `%` prefix and multi-line commands have `%%` prefix.
+
+**Database Commands**
+
+- `%listDatabases` - lists the databases on the database server.
+- `%whichDatabase` - which database am I connected to?
```

Comment:
Having a question here doesn't seem right. How about `returns the database name.`